packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2021 Red Hat One Platform
+ *
+ * MIT
+ *
+ * opc-feedback.ts enables the user to collect the feedback from the user
+ *
+ * @summary opc-feedback.ts will set the properties 
+ * @author Rigin Oommen
+ *
+ * Created at     : 2021-01-18 14:53:24 
+ * Last modified  : 2021-02-05 20:13:07
+ */
+
+import { LitElement, html, property, customElement, internalProperty, query } from 'lit-element';
+import style from './opc-feedback.scss';
+import { repeat } from 'lit-html/directives/repeat.js';
+import { defaultTemplate } from './defaultTemplate';
+@customElement('opc-feedback')
+export class OpcFeedback extends LitElement {
+  @property({ type: String, attribute: 'url' }) url;
+  @property({ reflect: true }) theme;
+  @property({ type: Object }) template = defaultTemplate;
+  @internalProperty()
+  openConfirmationModal = false
+  @internalProperty()
+  openFeedbackModal = false;
+  @internalProperty() protected summary = '';
+  @internalProperty() experience = '';
+  @internalProperty()
+  error = '';
+  errorList = [];
+
+  @query('textarea') textarea: HTMLTextAreaElement;
+
+  constructor() {
+    super();
+    this.theme = "red";
+    this.url = "/feedback";
+  }
+
+  static get styles() {
+    return [style];
+  }
+
+  resetForm() {
+    this.summary = '';
+    this.experience = '';
+    this.error = '';
+    this.textarea.value = '';
+  }
+
+  setExperience(selectedExperience) {
+    this.experience = selectedExperience;
+    this.errorList = this.template.experienceList.filter(experience => experience.name === this.experience)[0].errorList;
+  }
+
+  updateTemplate() {
```

Comment:
any function that shouldn't be used by the user should be prefixed with _ for eg: _updateTemplate(). please follow other components and how the conventions are used

Reply:
Updated the template and conventons as suggested